tests/integration/ha/continuous_writes.py

```diff
@@ -26,6 +27,14 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class ContinuousWritesResult:
```

Comment:
**praise** I see that you have been using data-classes, I really like this and makes function return very meaninfgul object which improve readability